packages/storybook/.storybook/manager-head.html

```diff
@@ -2,6 +2,8 @@
 <meta name="description"
   content="Azure Communication Services UI Library makes it easy for you to build modern communications user experiences using Azure Communication Services. It gives you a library of production-ready UI components that you can drop into your applications.">
 
+<meta name="google-site-verification" content="RLP3tg6OFhUU6_Q7EpOLm4GWjlBr3aYxkETkWLT6OfE" />
```

Comment:
Are we doing something similar for bing also?

Reply:
Thanks for the reminder. Added bing meta as well